_examples/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/machadovilaca/operator-observability/examples/metrics"
+)
+
+func main() {
+	metrics.SetupMetrics()
+
+	http.Handle("/metrics", promhttp.Handler())
+	go http.ListenAndServe(":2112", nil)
```

Comment:
maybe add here error handling

Reply:
this is just example to get it working, not need to handle the servers error I think